pizza_app/index.html

```diff
@@ -0,0 +1,134 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title>Task 12 Kottans - Pizza Queue</title>
+    <link rel="stylesheet" href="css/reset.css">
+    <link rel="stylesheet" href="css/style.css">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Lobster">
+    <script defer src="https://use.fontawesome.com/releases/v5.0.6/js/all.js"></script>
+</head>
+
+<body>
+    <header class="page-header">
+        <div class="current-time">
+            <i class="far fa-clock"></i>
+            <time datetime="2017-02-01T00:00:00">00:00:00</time>
+        </div>
+        <div class="title">
+            <img src="img/logo.svg" alt="logo">
+            <h1>Pizza Queue</h1>
+        </div>
+        <nav class="btn-wrapper">
+            <button class="btn btn-add" type="button">
+                <img src="img/add-icon.svg" alt="">
+            </button>
+            <button class="btn btn-login" type="button" aria-label="login / signup">
+                <i class="fas fa-sign-in-alt"></i>
```

Comment:
Here too? 🙂

Anyway, it looks quite strange.

Reply:
@viktor-yakubiv, `s` means `solid`

Reply:
@viktor-yakubiv yes, and `r` means `regular`. that's syntax from Font Awesome 5